proposals/0003-pr-process.md

```diff
@@ -0,0 +1,22 @@
+---
+author: Tikhon Jelvis
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/2
+---
+
+# Accepting Website PRs
+
+To quickly triage and accept PRs to the website, we need a process for reviewing, merging and deploying changes to the website. We adopted a process for this before we had this proposal repo, and the process helped us respond to and merge PRs faster than before. This proposal codifies that process to help make our policies explicit and transparent.
+
+Guidelines for accepting PRs to the website:
+
+  1. Purely technical fixes—like fixing the `default.nix` file, broken links, typos and HTML glitches—require one member of the committee to approve.
+  2. For uncontroversial changes that affect the content of the site, we need two committee members to approve. If the PR was authored by one or more committee members, it should be approved by *other* members unless there are less than two members not involved in the PR.
+  3. Major or controversial changes require a majority vote.
```

Comment:
Should we clarify major or controversial here? Ultimately it will be a matter of personal judgement but I think it might help to set expectations with some examples perhaps. 

Removing a link to a tool on the downloads page would be a good example to me of a controversial but not major change. 

Deprecating the wiki would be an example of a major but mostly in non-controversial change, but we could probably think of smaller examples as well.

Reply:
Adding some examples would be good but ultimately it will be up to the committee members what's major or controversial on a case by case basis, I don't think we can come up with a perfect definition here.

Reply:
Makes sense. I added a couple of examples but, ultimately, all of these rules come down to judgement calls one way or the other.